### PR TITLE
add bug to exempt labels on git stale.yml

### PR DESCRIPTION
**Pull request by lucasfcnunes**
- labels bug and wontfix don't go well together
- shutting down comments on bug issues is something intended?
- changing the stale label from wontfix to stale is less confusing. (wont)fix reminds of bugs. wontfix on e.g. questions and enhancements is kind of strange to me.
- plus... https://github.com/helmfile/helmfile/issues/893 (bug issue):
image
🤣

### CODE REVIEW

**Review by mumoshu — APPROVED** [review]
Ahh good catch 😂 
Yeah we should be able to just label something a `bug`, and it should remain open until it is unlabeled...
Thanks for your contribution @lucasfcnunes!

**Review by yxxhero — APPROVED** [review]

**Comment by lucasfcnunes** [comment]
> - changing the stale label from wontfix to stale is less confusing. (wont)fix reminds of bugs. wontfix on e.g. questions and enhancements is kind of strange to me.
could you guys give your thoughts on this one?

TL;DR I am proposing wontfix -> stale (or something else) as the stale label.

**Comment by mumoshu** [comment]
@lucasfcnunes Yeah maybe `stale` is more natural...? I loved `wontfix` for more explicitly saying it won't be fixed/addressed; stale is the cause of something getting considered `wontfix`; but yeah, `fix` doesn't make sense on anything other than bugs so...

WDYT @yxxhero?
